README.md

```diff
@@ -215,6 +204,8 @@ Cast times are currently only logged as you fish, so there won't be any data unt
 
 [See here](https://www.youtube.com/watch?v=GHgWIA-Zhug) for examples of the different tug types.
 
+Check [here](docs/Fisher-Troubleshooting.md) for common troubleshooting tips.
```

Comment:
I assume you mean docs/FAQ-TroubleShooting.md#fisher-module?

Reply:
Oops. Yes, that would be correct. Originally it was two files but it felt unnecessary. I've changed it.